Extensions/News/Compiler.js

```diff
@@ -10,6 +13,8 @@ class News {
         this.body = null;
 
         this._parseHeading(this.section, (heading) => {
+            // Nullptr means Null Pointer; In simple words the object reference was not set to an instance of an object.
+            // ctor<...> stands for constructor()
```

Comment:
let's use the full `constructor` - it faster to understand

Reply:
This file no longer exists, in the other PR